docs/content/python/new-roles/getting-started.md

```diff
@@ -191,83 +186,71 @@ uses [data filtering](../../guides/data_access) to translate the
+The `authorized_sessionmaker` requires as input: the Oso instance, the current user,
+and what permissions to check.
+
+In GitClub, we initialize the session on the application
 
 {{< literalInclude
     path="examples/gitclub-sqlalchemy-flask-react/backend/app/__init__.py"
-    lines="10,14,34-40,94"
+    from="docs: begin-authorized-session"
+    to="docs: end-authorized-session"
     gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
     >}}
 
-Before each request we fetch the logged in user and set the action
-depending upon the route.
+provide a helper decorator to define the permissions to check
+
+{{< literalInclude
+    path="examples/gitclub-sqlalchemy-flask-react/backend/app/routes/helpers.py"
+    from="docs: begin-session-decorator"
+    to="docs: end-session-decorator"
+    gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
+    >}}
 
-We can then issue regular SQLAlchemy queries to load authorized data:
+and we can then issue regular SQLAlchemy queries to load authorized data in our
+route handlers
 
 {{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/routes.py"
+    path="examples/gitclub-sqlalchemy-flask-react/backend/app/routes/repos.py"
     from="docs: begin-repo-index"
     to="docs: end-repo-index"
     gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
     linenos=true
     >}}
 
-`repos` will only contain repositories that `g.current_user` is allowed to take `g.current_action` on based on our policy.
-
-### Using the basic session
+### Making a query without authorization
 
-The basic session should be used for queries that should not have
-authorization applied. We create one using a typical SQLAlchemy `sessionmaker`.
+We can make queries without any authorization by setting the `checked_permissions=None`
+parameter. We pass this through in the `@session` decorator: 
 
 {{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/__init__.py"
-    lines="4,14,29,61"
+    path="examples/gitclub-sqlalchemy-flask-react/backend/app/routes/orgs.py"
+    from="docs: begin-is-allowed"
+    to="docs: end-is-allowed"
     gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
 >}}
 
-Often performing actions before authorization like authentication will use
-the basic session:
+In this case, we are still able to use the regular `oso.is_allowed` to authorize
+individual actions.
 
-{{< literalInclude
-    path="examples/gitclub-sqlalchemy-flask-react/backend/app/__init__.py"
-    from="docs: begin-authn"
-    to="docs: end-authn"
-    gitHub="https://github.com/osohq/gitclub-sqlalchemy-flask-react"
-    linenos=true
->}}
 
 ## Controlling access with roles
 
 Now, let's write our first rules that use role based access control. To
 setup the role library, we must:
 
-1. Initialize `OsoRoles`
-2. Persist role configuration to our database.
+1. Persist role configuration to our database.
```

Comment:
Fix numbers

Reply:
I'm a fan of the actual numbers lining up with the displayed numbers, but FWIW Markdown doesn't care what the actual numbers are. You could create a list as

```
1. one
1. two
1. three
```
and it'll display as

1. one
1. two
1. three